web/src/copy_and_paste.ts

```diff
@@ -292,7 +292,7 @@ function get_end_tr_from_endc($endc: JQuery<Node>): JQuery {
         // This handles the issue that Chrome seems to like selecting
         // the compose_close button when you go off the end of the
         // last message
-        return $(".message_row").last();
+        return rows.last_visible();
```

Comment:
Though the older code was not ideal, it just worked so far since the messages in the current narrow are always after the combined feed messages in the background, and so the last always happened to be the last visible